applications/crossbar/priv/couchdb/swagger/swagger.json

```diff
@@ -8299,6 +8321,26 @@
             "delete": {},
             "get": {}
         },
+        "/v2/accounts/{ACCOUNT_ID}/tasks": {
+            "get": {},
+            "put": {}
+        },
+        "/v2/accounts/{ACCOUNT_ID}/tasks/help": {
+            "get": {}
+        },
+        "/v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}": {
+            "delete": {},
+            "get": {},
+            "patch": {}
+        },
+        "/v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}/csv": {
+            "put": {}
+        },
+        "/v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}/csv/{ATTACHMENT_ID}": {
```

Comment:
Oh that shouldn't be there. @jamesaimonetti the generation of this files seems to be appending things? These endpoints are not available anymore